azurerm/internal/services/sentinel/sentinel_alert_rule_scheduled_resource.go

```diff
@@ -73,6 +73,24 @@ func resourceSentinelAlertRuleScheduled() *schema.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"event_grouping_setting": {
```

Comment:
I think we can remove setting
```suggestion
			"event_grouping": {
```

Reply:
updated